packages/wonder-blocks-button/components/button-core.js

```diff
@@ -86,25 +81,21 @@ export default class ButtonCore extends React.Component<Props> {
 
         if (href) {
             return router && !skipClientNav ? (
-                <StyledLink
-                    {...commonProps}
-                    onClick={this.handleClick}
```

Comment:
A few components were overriding the `onClick` callback after passing the handlers. I think this is from an earlier change where the default disabled `onClick` handler did something different. This was bad because the `onClick` prop was never called.

Verified that disabled components don't do anything.

Reply:
Is this what was causing `onClick` not to be called when `href` was set?

Reply:
Yeah.